src/main/scala/database/Queries.scala

```diff
@@ -19,4 +19,17 @@ object Queries {
     sql"""insert into prices (time_range, price)
     values ($time_range, $price)""".update
 
+  // Update user data
+  def updateSleepingTime(name: String, sleeping_time: Int) =
+    sql"""update users set sleeping_time = $sleeping_time where name = $name""".update
+
+  def updateWakeUpTime(name: String, wakeup_time: Int) =
+    sql"""update users set wakeup_time = $wakeup_time where name = $name""".update
+
+  def updateNightCharge(name: String, night_charge: Boolean) =
+    sql"""update users set night_charge = $night_charge where name = $name""".update
+
+  // Update devices data
```

Comment:
Split the queries into one file per table.
If more than one table is related(join) choose the main one

Reply:
all checks for all the queries in a file into one test and all the tests into a single file QueriesSuite test file